software/genmap/index/main.nf

```diff
@@ -0,0 +1,38 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process GENMAP_INDEX {
+    tag '$fasta'
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
+
+    conda (params.enable_conda ? "bioconda::genmap=1.3.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/genmap:1.3.0--h1b792b2_1"
+    } else {
+        container "quay.io/biocontainers/genmap:1.3.0--h1b792b2_1"
+    }
+
+    input:
+    path fasta
+
+    output:
+    tuple path(fasta), path("genmap_index/index.*"), emit: index
+    path "*.version.txt"                           , emit: version
```

Comment:
```suggestion
	path "genmap"       , emit: index
    path "*.version.txt", emit: version
```

Reply:
For consistency with other indexing modules e.g. [bwa index](https://github.com/nf-core/modules/blob/master/software/bwa/index/main.nf)